applications/sofa/gui/headlessRecorder/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+cmake_minimum_required(VERSION 3.1)
+project(SofaHeadlessRecorder)
+message("-- HeadlessRecorder is a 17.12 experimental feature, use it at your own risk.")
+
+
+set(HEADER_FILES
+    HeadlessRecorder.h)
+
+set(SOURCE_FILES
+    HeadlessRecorder.cpp)
+
+set(EXTRA_FILES
+    headlessRecorder_test
+)
```

Comment:
Is this usefull?

Reply:
From my point of view yes. By using this extra_files feature I can edit it from qtcreator and it is shown as part of my project

Reply:
But I can't see the `EXTRA_FILES` variable used anywhere. How does it have any impact on your project?

Reply:
By example, qtcreator will only display files which are parts of the project, HEADER_FILES and SOURCE_FILES. If you want to display a file which is not a source or header in your project tree then EXTRA_FILES is usefull. If you don't want to make this file visible from the project tree that's ok. But please, consider than headlessRecorder_test in not a binary file. It's a bash script.

Be able to edit it from the IDE is a plus. Consider it as a cpp test file, not as a binary.